pkg/controller/replicaset/metrics/metrics.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const ReplicaSetControllerSubsystem = "replicaset_controller"
+
+var DeletedPodAgeRatio = metrics.NewGauge(
+	&metrics.GaugeOpts{
+		Subsystem: ReplicaSetControllerSubsystem,
+		Name:      "deleted_pod_age_ratio",
+		Help:      "The ratio of chosen deleted pod's ages to the current youngest pod's age (at the time). Should be <2",
```

Comment:
I find this metric incredibly confusing..

Reply:
I agree, without reading your [comment](https://github.com/kubernetes/kubernetes/pull/101767/files#r657268235) on the other thread, I had no clue what this metric was supposed to be used for just from reading the help text.